python/lsst/ip/diffim/dcrModel.py

```diff
@@ -558,50 +562,46 @@ def applyImageThresholds(self, image, highThreshold=None, lowThreshold=None, reg
             image[lowPixels] = lowThreshold[lowPixels]
 
 
-def applyDcr(maskedImage, dcr, warpCtrl, bbox=None, useInverse=False, splitSubfilters=False):
+def applyDcr(image, dcr, useInverse=False, splitSubfilters=False, **kwargs):
     """Shift a masked image.
```

Comment:
It's no longer a masked image, and I think this docstring could be expanded: what direction is the shift, by how much, why is `dcr` a tuple?